packages/commonwealth/client/scripts/models/NotificationSubscription.ts

```diff
@@ -104,7 +102,7 @@ export const modelFromServer = (subscription: SubscriptionInstance) => {
 
   if (Thread) {
     try {
-      modeledThread = app.threads.modelFromServer(Thread);
+      modeledThread = new ThreadT(Thread as any);
```

Comment:
Unfortunately, here we have to use it. There are 2 models involved
- [the one from api](https://github.com/hicommonwealth/commonwealth/blob/551dc67478e75a560b5c00f3ef59bd731aad988f/packages/commonwealth/server/models/thread.ts)
- [the one in frontend](https://github.com/hicommonwealth/commonwealth/blob/e5e15d551c918451ef5a05c84bcee27570a06af2/packages/commonwealth/client/scripts/models/Thread.ts)

here the type of `Thread` variable is of the API model and we are modeling it to the frontend model -- both have incompatible types.

The old `app.threads. modelFromServer()` was simply ignoring the extra/missed fields, which we are also ignoring now. But since the thread model has typescript args its complaining about lint. So I added any here.

I will add a comment to code.

Reply:
We could come up with a new transformer directory, but I think that could be a followup for the whole models directory after all of it is cleaned up, what do you suggest?

Reply:
yeah, I do not insist to work on it in this PR, but definitely it would be nice to tidy it up so we can use types with confident. Using typescript is pointless when we have to throw `any`s here and there.

Reply:
[Followup here](https://github.com/hicommonwealth/commonwealth/pull/4588#issuecomment-1665613964)